scripts/sysman/memoryBwCounterValidBits.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Bandwidth Counter Valid Bits"
+version: "1.8.1"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Memory Bandwidth Counter Valid Bits Extension Name"
+version: "1.8.1"
+name: $S_MEMORY_BW_COUNTER_VALID_BITS_EXP_NAME
+value: '"$XS_extension_mem_bw_counter_valid_bits"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Memory Bandwidth Counter Valid Bits Extension Version(s)"
+version: "1.8.1"
+name: $s_mem_bw_counter_bits_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties for reporting valid bit count for memory bandwidth counter value"
+version: "1.8.1"
+class: $sMemory
+name: $s_mem_bw_counter_bits_exp_t
+details:
+    - "Number of valid counter bits"
```

Comment:
Can we mention read and write counter bits of memory bandwidth, just to be a little specific ?

Reply:
done